lib/wibox/container/background.lua

```diff
@@ -30,8 +31,11 @@ end
 
 -- Prepare drawing the children of this widget
 function background:before_draw_children(context, cr, width, height)
+    local bw    = self._private.shape_border_width or 0
```

Comment:
I bet luacheck complains that `bw` is unused.

Reply:
It is used in the line directly below.